batchflow/models/tf/layers/block.py

```diff
@@ -39,6 +40,7 @@
     'c': 'conv',
     't': 'transposed_conv',
     'C': 'separable_conv',
+    'w': 'depthwise_conv',
```

Comment:
Let's make it `W` to align with separable `C` and `T`

Reply:
done.